resources/public/api/conf/P1.0/examples/contact-details-response.json

```diff
@@ -0,0 +1,17 @@
+{
+  "_links": {
+    "self": {
+      "href": "/individuals/details/contact-details?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430"
+    }
+  },
+  "contactDetails": [
+    {
+      "type": "MOBILE TELEPHONE",
+      "detail": "07123 987654"
+    },
+    {
+      "type": "PRIMARY E-MAIL",
+      "detail": "fred.blogs@hotmail.com"
```

Comment:
We should not be outputting an email. We need to output telephone numbers only (all three types)